imports/wasi_snapshot_preview1/fs_test.go

```diff
@@ -8,8 +8,9 @@ import (
 	"io/fs"
 	"math"
 	"os"
-	"path"
+	pathutil "path"
```

Comment:
I prefer not to use alias..

Reply:
we do this in many files, and we do this because we have fields literally named path in specs. do you want me to revert all of them or just here?

Reply:
the other impact is it forces us to choose dumb names like pathName etc.

Reply:
ok fine.

Reply:
I think on a whole things are worse, but doing it.

Reply:
done